pkg/systemd/services/services.jsx

```diff
@@ -471,7 +475,7 @@ class ServicesPageBody extends React.Component {
 
                                         return this.getUnitByPath(unit_path);
                                     }, ex => {
-                                        this.props.isLoading(false);
+                                        this.props.setIsLoading(false);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18411-20230227-080740-f45e3de4-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)